lib/Basic/CmdLineOptions.cpp

```diff
@@ -210,7 +210,12 @@ WPInterpolant("wp-interpolant",
 llvm::cl::opt<bool>
 MarkGlobal("mark-global",
            llvm::cl::desc("Decide whether global variables are marked or not"),
-           llvm::cl::init(false));
+           llvm::cl::init(true));
+
+llvm::cl::opt<bool>
+SpecIndependence("-spec-independence",
```

Comment:
This is related to spec-independence. Please move to another PR.

Reply:
This is fixed!